### PR TITLE
Add edgeworker release notes for release 2024.2.400

### PR DESCRIPTION
**Pull request by jp4a50**
Opening this up now to get eyes on it and amend the release notes.

We shouldn't merge until edgeworker-debug has been released.

### CODE REVIEW

File: data/changelogs/workers.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 link: "/workers/platform/changelog/"
 productName: Workers
 entries:
+- publish_date: '2024-02-22'
+  description: |-
+    - Sockets now support an `opened` attribute.
```
Comment (jp4a50) [inline data/changelogs/workers.yaml]: @dom96

File: data/changelogs/workers.yaml [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+    - Set a new timeout for alarm handlers.
```
Comment (jp4a50) [inline data/changelogs/workers.yaml]: @jqmmes is it increased? decreased? How does this affect users?

File: data/changelogs/workers.yaml (hunk 1 above)
Comment (irvinebroque) [reply data/changelogs/workers.yaml]: Link to relevant docs section?

Comment (dom96) [reply data/changelogs/workers.yaml]: Best thing we can link to is: https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/#socket

**Review by dom96 — APPROVED** [review]

File: data/changelogs/workers.yaml (hunk 2 above)
Comment (irvinebroque) [reply data/changelogs/workers.yaml]: Should we put fix to starting tail/log stream <> DO here? @jp4a50 @vy-ton

Comment (jp4a50) [reply data/changelogs/workers.yaml]: Yep, I think so.

Edit: as pointed out by Taylor, this behaviour isn't rolled out to all users yet, so we'll have to hold off for now.

Comment (jqmmes) [reply data/changelogs/workers.yaml]: The timeout was decreased from 30 minutes to 15 minutes.

Comment (jp4a50) [reply data/changelogs/workers.yaml]: @jqmmes why? And what do we want to tell our users about this change, if anything? The current release note doesn't look helpful to users.

Comment (irvinebroque) [reply data/changelogs/workers.yaml]: @vy-ton @joshthoward ^

Comment (jqmmes) [reply data/changelogs/workers.yaml]: Alarm handlers were allowed to run forever. We introduced a maximum handler wall time of 15 minutes.
Maybe we can phrase as:
"Set a maximum wall time for alarm handlers with 15 minutes default timeout"

File: data/changelogs/workers.yaml [hunk 3]
```diff
@@ -2,6 +2,10 @@
 link: "/workers/platform/changelog/"
 productName: Workers
 entries:
+- publish_date: '2024-02-23'
+  description: |-
+    - Sockets now support an [`opened`](/workers/runtime-apis/tcp-sockets/#socket) attribute.
+    - [Durable Object alarm handlers](/durable-objects/api/alarms/#alarm) now impose a maximum wall time of 15 minutes.
```
Comment (deadlypants1973) [inline data/changelogs/workers.yaml:8]:
```suggestion
    - [Durable Object alarm handlers](/durable-objects/api/alarms/#alarm) now impose a maximum wall-clock time of 15 minutes.
```

**Review by deadlypants1973 — APPROVED** [review]

File: data/changelogs/workers.yaml (hunk 3 above)
Comment (deadlypants1973) [reply data/changelogs/workers.yaml:8]: Assuming the definition being used is wall-clock time: /workers/platform/limits/#duration

File: data/changelogs/workers.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@ entries:
 - publish_date: '2024-02-23'
   description: |-
     - Sockets now support an [`opened`](/workers/runtime-apis/tcp-sockets/#socket) attribute.
-    - [Durable Object alarm handlers](/durable-objects/api/alarms/#alarm) now impose a maximum wall time of 15 minutes.
+    - [Durable Object alarm handlers](/durable-objects/api/alarms/#alarm) now impose a maximum wall-clock time of 15 minutes.
```
Comment (deadlypants1973) [inline data/changelogs/workers.yaml]:
```suggestion
    - [Durable Object alarm handlers](/durable-objects/api/alarms/#alarm) now impose a maximum wall time of 15 minutes.
```

**Comment by irvinebroque** [comment]
Merging this since this is from awhile back and we need to get release notes out.

**Pull request merged**